crates/symbolicator-js/src/service.rs

```diff
@@ -44,8 +45,11 @@ impl SourceMapService {
             in_memory,
         ));
 
+        let files_in_bundles = FileInBundleCache::new();
+
         Self {
             objects,
+            files_in_bundles,
```

Comment:
Since we do the same thing on line 54:

```suggestion
            files_in_bundles: FileInBundleCache::new(),
```